apps/web/src/components/Onboarding/ModalBackButton.tsx

```diff
@@ -3,7 +3,18 @@ import { IconButton, type IconButtonProps } from "@chakra-ui/react";
 import { ArrowLeftCircleIcon } from "../../assets/icons";
 import { useColor } from "../../styles/useColor";
 
-export const ModalBackButton = (props: IconButtonProps) => {
+export const ModalBackButton = (props: Omit<IconButtonProps, "aria-label">) => {
```

Comment:
looks like this button will always have `onClick={goBack}` ? maybe it's beneficial to inline it here?